src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,328 @@
+                        <View style={[styles.peopleRowCell, styles.flex4]}>
+                            <Text style={[styles.textStrong]}>
+                                {this.props.translate('common.email')}
+                            </Text>
+                        </View>
+                    )
+                }
+                <View style={[styles.peopleRowCell, styles.flex1]} />
+            </View>
+        );
+    }
+
+    render() {
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title={this.props.translate('common.people')}
+                    onCloseButtonPress={() => Navigation.dismissModal(true)}
+                />
+                <ConfirmModal
+                    danger
+                    title={this.props.translate('workspace.people.removeMembersTitle')}
+                    isVisible={this.state.isRemoveMembersConfirmModalVisible}
+                    onConfirm={this.removeUsers}
+                    onCancel={this.hideConfirmModal}
+                    prompt={this.props.translate('workspace.people.removeMembersPrompt')}
+                    confirmText={this.props.translate('common.remove')}
+                    cancelText={this.props.translate('common.cancel')}
+                />
+                <ScrollView style={[styles.settingsPageBackground]} bounces={false}>
+                    <View style={styles.pageWrapper}>
+                        <Text style={[styles.mb6, styles.textP, styles.textFull]}>
+                            {this.props.translate('workspace.people.tagline')}
+                        </Text>
+                        <View style={styles.buttonRow}>
+                            <Button
+                                success
+                                style={[]}
+                                text={this.props.translate('common.invite')}
+                                onPress={this.inviteUser}
+                            />
+                            <Button
+                                danger
+                                style={[styles.ml2]}
+                                isDisabled={this.state.selectedEmployees.length === 0}
+                                text={this.props.translate('common.remove')}
+                                onPress={this.askForConfirmationToRemove}
+                            />
+                        </View>
+                        <View style={[styles.w100, styles.mt4]}>
+                            {
+                                this.props.policy.employeeList ? (
+                                    <FlatList
+                                        ListHeaderComponent={this.renderHeader()}
+                                        ref={flatListRef}
+                                        renderItem={this.renderItem}
+                                        contentContainerStyle={[styles.w100]}
+                                        initialRowHeight={32}
```

Comment:
This seems unnecessary

Reply:
The container style needs to be `width: 100%` otherwise the columns do not align well.